plugins/module_utils/foreman_helper.py

```diff
@@ -546,7 +546,13 @@ def _resource_call(self, resource, *args, **kwargs):
         return self._resource(resource).call(*args, **kwargs)
 
     def _resource_prepare_params(self, resource, action, params):
-        return self._resource(resource).action(action).prepare_params(params)
+        api_action = self._resource(resource).action(action)
```

Comment:
previously, this swallowed all params that need to be passed as part of the URL instead of GET/POST data.

Reply:
and yes, I checked, this does not break the fix for #977

Reply:
I wonder if the real fix should live in apypie tho